pkg/daemon/daemon.go

```diff
@@ -188,6 +189,10 @@ func (d *Daemon) WaitUntilState(expected types.DaemonState) error {
 					d.ID(), expected, state)
 			}
 
+			if exportErr := exporter.RecordDaemonEvent(d.ID(), string(expected)); exportErr != nil {
```

Comment:
Can we rename exporter to collector, which means the metrics is not exported to other system yet

Reply:
Yes, finished.